site/content/docs/5.3/examples/dashboard/index.html

```diff
@@ -2,6 +2,7 @@
 layout: examples
 title: Dashboard Template
 extra_css:
+  - "https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.3/font/bootstrap-icons.css"
   - "dashboard.css"
 extra_js:
   - src: "https://cdn.jsdelivr.net/npm/feather-icons@4.28.0/dist/feather.min.js"
```

Comment:
Since we are now using Bootstrap icons in this file (and in the corresponding RTL one), I think we can remove this `feather.min.js` in both files.